### PR TITLE
Shared Storage landing page

### PR DESCRIPTION
**Pull request by heyawhite**
Privacy Sandbox docs

### CODE REVIEW

File: site/en/docs/privacy-sandbox/shared-storage/index.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+const opaqueUrl = await window.sharedStorage.runURLSelectionOperation(
+  'select-url', [defaultUrl, adUrl]
+);
+
+// Render the returned URL into an iframe or a fenced frame
+document.getElementById('example-iframe').src = opaqueUrl;
+```
+
+
+### Example: aggregation of cross-site data (ad campaign reach) {: #aggregated-data }
```
Comment (kevinkiklee) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:157]: Just leaving a reaction:  Oh, I didn't realize that the explainer example was updated: https://github.com/pythagoraskitty/shared-storage/commit/8ec100d7ee0dae1c52fb0a53f040b30af89053f9.  Yea, this example is a much better example.  Way better than what I attempted to come up with.

File: site/en/docs/privacy-sandbox/shared-storage/index.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+values in shared storage, and decrements the value with each additional view.
+If there are available impressions left (the user has not hit their frequency
+cap), the ad is returned (index `1`). If not, the default URL is returned
+(index `0`).
+
+In this example, the adtech creates two files:
+
+*  `get-url.js` is the shared storage worklet, which evaluates the frequency
+   cap to determine which URL is returned for the ad creative.
+*  `main.js` selects and renders the ad creative and is injected on a
```
Comment (kevinkiklee) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:105]: I like how you have `ad-iframe.js` in the second example.  Let's update this to `ad-iframe.js` also to match the second example.

Comment (kevinkiklee) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:105]: I like how you have `ad-iframe.js` in the second example.  Let's update this to `ad-iframe.js` also to match the second example.

**Review by kevinkiklee — DISMISSED** [review]

File: site/en/docs/privacy-sandbox/shared-storage/index.md (hunk 2 above)
Comment (heyawhite) [reply site/en/docs/privacy-sandbox/shared-storage/index.md:105]:
```suggestion
*  `ad-iframe.js` selects and renders the ad creative and is injected on a
```

File: site/en/docs/privacy-sandbox/shared-storage/index.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+  }
+}
+
+// Register the operation
+registerURLSelectionOperation(
+   'select-url', SelectURLOperation
+);
+```
+
+**main.js**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:134]:
```suggestion
**ad-iframe.js*
```

File: site/en/docs/privacy-sandbox/shared-storage/index.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+(index `0`).
+
+In this example, the adtech creates two files:
+
+*  `get-url.js` is the shared storage worklet, which evaluates the frequency
+   cap to determine which URL is returned for the ad creative.
+*  `main.js` selects and renders the ad creative and is injected on a
+   publisher's website.
+
+**get-url.js**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:108]:
```suggestion
#### `get-url.js`
```

File: site/en/docs/privacy-sandbox/shared-storage/index.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+multiple sites, to maximize the number of people who see their ad. The company
+wants to know their reach, how many unique users have seen the ad. In this
+example, the publisher creates two files: 
+
+*  `ad-iframe.js` should be embedded in the iframe, which calls to the shared
+   storage worklet and sends a report with the ad campaign ID.
+*  `reach.js`  is the shared storage worklet which sends the reach report to
+   be aggregated.
+
+**ad-iframe.js**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:172]:
```suggestion
#### `ad-iframe.js`
```

File: site/en/docs/privacy-sandbox/shared-storage/index.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+// Send a report to the shared storage worklet
+await window.sharedStorage.runOperation("send-reach-report", {
+  data: {
+     "campaign-id": "1234"
+  }
+});
+```
+
+**reach.js**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/shared-storage/index.md:185]:
```suggestion
#### `reach.js`
```

**Pull request merged**